cellmlmanip/parser.py

```diff
@@ -68,16 +68,13 @@ def __init__(self, name):
 
     def set_parent(self, parent_name):
         """Sets the parent of this component"""
-        if self.parent:
-            raise ValueError('Parent of component %s already %s. Cannot set %s!' % (self.name,
-                                                                                    self.parent,
-                                                                                    parent_name))
+        assert not self.parent, 'Parent of component %s already %s. Cannot set %s!' % (self.name, self.parent,
```

Comment:
Do we want an assert here? I think it depends on whether the CellML is pre-validated or not. If the CellML is valid, this should never happen, so an assert is fine. If calling this method is part of how we validate the CellML, then this should be a user error, that says something like "Multiple parents specified for component X".
Same for `add_sibling` below.

What do you think, @jonc125 ?

Reply:
@MichaelClerx  Ok actually I finally found a way to trigger this with wrong cellml, so I'll reinstate the raise error and add a test

Reply:
@MauriceHendrix  Ok I found a way to trigger this, I'll revert + test